lib/ncr_dispatcher.rb

```diff
@@ -0,0 +1,6 @@
+class NcrDispatcher < LinearDispatcher
```

Comment:
Open to doing away with this altogether and maybe injecting this logic somehow into `requires_approval_notice?` in LinearDispatcher.